File: app/src/main/java/com/imdc/milkdespencer/enums/UserTypeEnum.java
package com.imdc.milkdespencer.enums;

public enum UserTypeEnum {
    ADMIN(0),

    END_USER(1),

    CUSTOMER_ADMIN(2);

    private final int userTypeValue;

    // Constructor
    UserTypeEnum(int userTypeValue) {
        this.userTypeValue = userTypeValue;
    }

    // Getter method
    public int value() {
        return userTypeValue;
    }
}

File: app/src/main/java/com/imdc/milkdespencer/enums/UserTypeEnum.kt
package com.imdc.milkdespencer.enums

enum class UserTypeEnum // Constructor
    (private val userTypeValue: Int) {
    ADMIN(0),
    END_USER(1),
    CUSTOMER_ADMIN(2);

    // Getter method
    fun value(): Int {
        return userTypeValue
    }
}
